lib/generators/deimos/schema_class_generator.rb

```diff
@@ -197,6 +230,106 @@ def _field_type(avro_schema)
       def _schema_base_class(avro_schema)
         Deimos::SchemaBackends::AvroBase.schema_base_class(avro_schema)
       end
+
+      # An ERB template for schema record classes
+      # @return [String]
+      def _record_class_template
```

Comment:
erm please put this in a file and read it in rather than pasting the code in the generator?

Reply:
you sir are smart!